app/models/health/patient.rb

```diff
@@ -407,14 +407,20 @@ def self.cp_2_engagement(on) # rubocop:disable Naming/MethodParameterName
     end
 
     scope :intake_required, -> do
-      where.not(id: Health::PctpCareplan.sent.select(:patient_id))
+      where.not(id: Health::PctpCareplan.sent.pluck(:patient_id)).
+        or(where(id: where.missing(:pctp_careplans).pluck(:id)))
```

Comment:
I don't think you're crossing database boundaries here, no need to pluck.
```suggestion
      where.not(id: Health::PctpCareplan.sent.select(:patient_id)).
        or(where(id: where.missing(:pctp_careplans).select(:id)))
```

Reply:
I think there was a problem with `or`s not having the same joins that had me doing this -- but I maybe I resolved this -- they seem like the same scope now.